frontend/src/organizations/components/OrganizationEditForm.js

```diff
@@ -1,35 +1,35 @@
 /*
  * Presentational component
  */
-import React from 'react'
-import PropTypes from 'prop-types'
-import FontAwesomeIcon from '@fortawesome/react-fontawesome'
+import React from "react";
```

Comment:
check your linting setup for TFRS because we use JS standard which doesn't use semi-colons.

Reply:
Thank you for pointing this out! The linting issue is now resolved. I'll keep this in mind for next time.